### PR TITLE
feat: corrected-short-answer component

### PR DESCRIPTION
**Pull request by jfdoming**
## Notion ticket link

[Short Answer component](https://www.notion.so/uwblueprintexecs/Short-Answer-component-3b0fd67fa1764c2399d63182de80d215)

## Implementation description
- Add corrected-answer helper components
- Add corrected-short-answer component
- Add demo of short-answer component to component library

image

## Steps to test
1.

## What should reviewers focus on?
* Have the helper components been generified correctly?

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by jfdoming** [comment]
Once this PR is merged I expect merging 2 of the other component types to be fairly fast. The fraction component will probably have to wait until we finalize a data model for it.

File: frontend/src/components/sessions/results/CorrectedShortAnswer.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  correctAnswer: number;
+};
+
+const CorrectedShortAnswer = ({
+  studentAnswer,
+  correctAnswer,
+}: CorrectedShortAnswerProps) => (
+  <CorrectedInputWrapper
+    isWrongAnswer={studentAnswer !== correctAnswer}
+    w="34%"
```
Comment (joyce-shi) [inline frontend/src/components/sessions/results/CorrectedShortAnswer.tsx:17]: LOL

Comment (joyce-shi) [inline frontend/src/components/sessions/results/CorrectedShortAnswer.tsx:17]: LOL

**Review by joyce-shi — APPROVED** [review]
some _goooood looking_ short answer components 😍

**Pull request merged**